tests/fixtures/site.py

```diff
@@ -1,41 +1,150 @@
+import math
+from datetime import datetime, timedelta
+
 import pytest
 
+from django.utils import timezone
 
-@pytest.fixture
-def site_matrix(factories):
+
+def _add_project_resources(project):
+    return [
+        dict(project=project, total_strings=(i + 5) ** 3)
+        for i in range(0, 7)]
+
+
+def _add_project_locales(locales, project):
+    return [
+        dict(project=project, locale=locale)
+        for locale in locales]
+
+
+def _add_translated_resources(locales, project, resources):
+    _kwargs = []
+    for i, resource in enumerate(resources):
+        for locale in locales:
+            _kwargs.append(
+                dict(locale=locale,
+                     approved_strings=(i + 2) ** 3,
+                     translated_strings=(i + 1) ** 3,
+                     fuzzy_strings=i ** 3))
+    return _kwargs
```

Comment:
These three functions are called `_add_something` but I don't see them adding anything. Shouldn't they be called `_get_something` instead, as they return the items you're interested in?

Reply:
yep, renamed